### PR TITLE
Instructions on GitHub & Markdown page (fixes issue #1255)

### PR DESCRIPTION
**Pull request by jwill12**
https://rawgit.com/jwill12/jwill12.github.io/iss4/#!pages/githubandmarkdown.md

### CODE REVIEW

**Comment by jwill12** [comment]
new commits fix the issue where website link was highlighted incorrectly

File: pages/githubandmarkdown.md [hunk 1]
```diff
@@ -66,15 +66,11 @@ Don't worry if you see a **404** when you access the link, it will take a while
 
 **NOTE**: Make sure you are looking at the forked repo instead of the original Open Learning Exchange repo before working on it. Forks let you make changes to a project without affecting the original repository. You can fetch updates from or submit changes to the original repository with pull requests.
 
-
-### Create a new file as your personal MDwiki page and commit your changes
- Before creating your file create a new branch using the branch drop down menu as shown below. Click on the Branch drop down menu and name your branch(you can give any name to your branch) then click Create branch. Once you are done with that now you can see that your branch is listed in the menu.
```
Comment (xyb994) [inline pages/githubandmarkdown.md]: Why are we removing this paragraph?

Comment (jwill12) [reply pages/githubandmarkdown.md]: Sync issue I believe with between the master & created branch. made the changes cause @kylemathias requested for them. should be all set now

Comment (xyb994) [reply pages/githubandmarkdown.md]: @jwill12 The problem is that now the heading and paragraph are gone in your branch compare with master.

Refer to:
https://help.github.com/articles/resolving-a-merge-conflict-on-github/
https://github.com/blog/2293-resolve-simple-merge-conflicts-on-github

File: pages/githubandmarkdown.md [hunk 2]
```diff
@@ -52,29 +52,27 @@ Now, you will fork the repository to your own account. Look at the picture below
...
 
 ### Go to Settings and rename your repository
-After you fork the repository, you will be on your page: &lt;username&gt;/open-learning-exchange.github.io. Then, you should go to Settings at the top middle of the page and rename your repository to your GitHub username. See example below.
+After you fork the repository, you will be on your page: `username/open-learning-exchange.github.io`. Then, you should go to **Settings** at the top middle of the page and rename your repository to your GitHub username. See example below.
 
 ![Renaming Repository](uploads/images/Rename Repository.png)
 
 ### Post a link to your repository in chat
-After you rename it, make sure to post a link to the github.io website you've just created (&lt;username&gt;.github.io) in the [gitter.im chat](https://gitter.im/open-learning-exchange/chat).
+After you rename it, make sure to post a link to the github.io website you've just created `username.github.io` in the [gitter.im chat](https://gitter.im/open-learning-exchange/chat).
```
Comment (xyb994) [inline pages/githubandmarkdown.md]: The use of `username.github.io` seem to be inconsistent with [line 44](https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/1256/files#diff-e91ae61c4ef311b20ddd7906072e1cf6R44) now.

**Comment by kylemathias** [comment]
Looks like this never got merged and it now has Conflicting files. Fix the conflicts and let us know so we can go and merge it.

**Comment by xyb994** [comment]
Since this PR is stale, changes will be incorporated into #1547.
Closes #1255.

**Pull request closed without merging**